splink/connected_components.py

```diff
@@ -295,7 +295,10 @@ def _cc_assess_exit_condition(representatives_name):
 
 
 def _cc_create_unique_id_cols(
-    linker: "Linker", concat_with_tf: str, df_predict: str, match_probability_threshold
+    linker: "Linker",
+    concat_with_tf: str,
+    df_predict: str,
+    match_probability_threshold,
```

Comment:
since you're editing this line already, can you add a type hint?

```suggestion
    match_probability_threshold: float,
```